lang/en/auth.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    'failed'   => 'These credentials do not match our records.',
```

Comment:
```suggestion
    'failed' => 'These credentials do not match our records.',
```
fix tương tự các line khác.